website/pages/chapters/assessing_performance.js

```diff
@@ -255,31 +314,45 @@ export default function AssessingPerformance() {
         <p>
           Well in some sense, you want as big of a test set as possible since
           the more examples in your test set, the better estimate of the true
-          error you will get. You can think of the extreme case where your test
-          set contains all possible input/outputs, then you will exactly be able
+          error you will get. Imagine the extreme case where the test set
+          contains all possible input/outputs, then you will exactly be able
+          <MatthewComment>
+            This is not necessarily true? I think you want your test set to
+            represent the underlying joint distribution{" "}
+            <IM math={`(x,y) \\sim P(x, y).`} />
+          </MatthewComment>{" "}
           to compute the true error.
         </p>
 
         <p>
-          However, by making your test set larger, you will need to be making
-          your training set smaller. This can cause problems since we want as
-          much training data possible to give us the best possible estimate of
-          the true function. Consider the animation below that compares a small
-          training dataset to a large one.
+          However, because you only have finite data, by making your test set
+          larger, you will need to comprimise be making your training set
+          smaller. This can cause problems where the ML Algorithm fails to
+          capture the underlying structure due to lack of data. Consider the
+          animation below that compares a small training dataset to a large one.
+        </p>
+
+        <p>
+          TODO(manim): show how training/test distribution effects learner
+          performance.
         </p>
 
         <p>
-          In practice, people generally use a ratio of 80% train and 20% test or
-          90% train and 10% test, but this really depends on your context and
-          how much data you have! Two very important points about this test set:
+          ML practitioners generally use a ratio of 80% train/20% test or 90%
+          train/10% test, but this really depends on the context and how much
+          data you have! It's up to you to understand the problem and how to
+          allocate this data. Two very important points to keep in mind when
+          selecting the test set:
         </p>
 
         <ul>
           <li>
-            When splitting a train/test set, you should do so randomly. If you
-            selected the last 20% of the data as a test set, you could
-            potentially introduce biases in the test set if your data was
-            originally sorted, say by square footage.
+            When splitting a train/test set, you should (must?) do so randomly.
```

Comment:
Not must. For time series analysis, you generally want your test data to come after your train data chronologically.

Reply:
:+1: Assumed we were only talking about iid re linear regression